schemas/subs/vc-common.js

```diff
@@ -0,0 +1,121 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {config} = require('bedrock');
+const {schemas} = require('bedrock-validation');
+
+const {constants} = config;
+
+const proof = () => ({
+  type: 'object',
+  title: 'JSON LD Proof',
+  required: [
+    'type',
+    'created',
+    'proofPurpose',
+    'verificationMethod',
+    'jws'
+  ],
+  properties: {
+    type: {type: 'string'},
+    proofPurpose: {
+      type: 'string',
+      examples: [
+        'authentication',
+        'capabilityDelegation'
+      ]
+    },
+    verificationMethod: {type: 'string'},
+    created: schemas.w3cDateTime(),
+    challenge: {type: 'string'},
+    jws: {type: 'string'}
+  }
+});
+
+// add the capabilitychain for the CapabilityDelegationProof.
+const capabilityDelegationProof = () => {
+  const _proof = proof();
+  _proof.title = 'Capability Delegation Proof';
+  _proof.required.push('capabilityChain');
+  _proof.properties.capabilityChain = {
+    type: 'array',
+    items: {type: 'string'},
+    minItems: 1
+  };
+  _proof.properties.proofPurpose.const = 'capabilityDelegation';
+  return _proof;
+};
+
+const allowedActionArray = {
+  title: 'Allowed Action Array',
+  type: 'array',
+  items: {type: 'string'},
```

Comment:
```suggestion
  items: allowedActionString,
```
and move allowedActionString above.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-vc-issuer/pull/20/commits/3234cbca6753941fc2e1350f630102a13def425c